e2e/src/pages/commentspanel.po.ts

```diff
@@ -0,0 +1,103 @@
+import {browser, by, element, ElementArrayFinder, ElementFinder,} from 'protractor';
+import {AppPage} from './app.po';
+import {By} from '@angular/platform-browser';
+import {expect} from 'chai';
+
+
+export class CommentsPanelPage extends AppPage {
+  //commentsToggleButton : ElementFinder = element(by.id('toggle-comments-panel'));
+
+  commentsToggleButton : ElementFinder      = element(by.xpath("//button[@id=\'toggle-comments-panel\' and @title=\'Toggle Comments Panel\']"));
+  commentsTabButton     : ElementFinder     = element(by.xpath(("//div[@class=\'govuk-tabs commentSummaryHeader\']/ul/li[1]/a")));
+  filterTabButton       : ElementFinder     = element(by.xpath(("//div[@class=\'govuk-tabs commentSummaryHeader\']/ul/li[2]/a")));
+  searchTabButton       : ElementFinder     = element(by.xpath(("//div[@class=\'govuk-tabs commentSummaryHeader\']/ul/li[3]/a")));
+  collateCommentsButton : ElementFinder     = element(by.xpath("//button[@id=\'commentSummary\']"));
+  commentsNotAvailable  : ElementFinder     = element(by.xpath("//div[@id=\'comment-container\'/h3"));
+  courtBundleName       : ElementFinder     = element(by.xpath("//div[@id=\'comment-container\'/h2"));
+  overlayCloseButton    : ElementFinder     = element(by.buttonText("Close"));
+  searchInputField      : ElementFinder      = element(by.id('search-comments-input'));
+  searchButton          : ElementFinder     = element(by.xpath('//button[@class=\'govuk-button comment-search\']'));
+  searchResult          : ElementFinder     = element(by.css('#comment-search'));
+
+  async clickShowCommentsToggleIcon(){
+     await this.commentsToggleButton.click();
```

Comment:
I think naming this function clickShowCommentsToggleIcon is misleading, because it might show or hide depending on the current state

Reply:
Renamed to clickCommentsToggleIcon